plugins/api-docs/src/components/ApisCards/HasApisCard.tsx

```diff
@@ -76,9 +76,9 @@ export const HasApisCard = ({ variant = 'gridItem' }: Props) => {
       variant={variant}
       emptyContent={
         <div>
-          No API is part of this system.{' '}
+          This {entity.kind.toLowerCase()} does not contain any APIs.{' '}
```

Comment:
I think whether we want to template this is a general design question for components in Backstage. We could make them very flexible and reusable in every case (with a lot of configuration and complexity) or make it easy do write custom variants of a component (more overhead by copy & paste). Right now the HasApisCard is just a thin layer around the entity table that contains the actual logic.

But I think for now it's the easiest route to go if you have such a usecase

Reply:
That's a good point.

* https://backstage.io/docs/features/software-catalog/system-model#docsNav
  * Here we say "A system, in this sense, is a collection of resources and components that exposes one or several public APIs..
  * So the verb is *exposes*.
* https://backstage.io/docs/features/software-catalog/descriptor-format#kind-system
  * Here we say, "The system may expose or consume one or several APIs."
  * So again "exposes".

Maybe exposes? Or maybe we should align back to provides to that doco here as well...

Reply:
Exposing APIs from a system is a different functionality that isn't implemented. The idea is that an API is exposed for usage by other systems. PartOf is the relationship that is in use here.

Reply:
I may have missed that nuance.  Are we saying that today in the Backstage model an API could be provided (`providesApi`), but not exposed (`partOf`), so the implication no one could actually call it from outside of itself?

`<joke/half serious>` Wouldn't that just be called a private function? `</joke/half serious>` 🤔

Reply:
Or are we talking boundary of a system, like a microservice in a system could call another microservice API in the same system, but when you say expose you mean expose OUTSIDE of the boundary of a system?

Reply:
So what is possible right now:

* Components can **provide** an API (which is a separate entity)
* Components can be **part of** a System.
* APIs (on their own) can be **part of** a System. For me that wouldn't mean that it is "exposed" or something like that, this is only a grouping.

Not possible:
* Systems can **expose/provide/make public** an API.

> OUTSIDE of the boundary of a system

Yes. It's about discoverability. Other teams responsible for other parts of a bigger picture might want to reuse existing things from other systems. But I think, at least personally, that an API that is reused by other people should have other "standards" especially when it comes to compatibility compared to team internal APIs.

As a small background about the initial wording: I chose to go very close to the wording of the relations. Beside that I later made a copy and paste mistake 😆 . I had an issue with the HasPart relationships as it wasn't that easy to add here compared to PartOf. That's the reason why I went with "contains".